lib/redux/actions/async.js

```diff
@@ -538,13 +582,37 @@ export function retrieveTargetsFromStorage() {
   };
 }
 
+export function createCustodialAccount(profile) {
+  return (dispatch, getState) => {
+    const { api } = services;
+    dispatch(syncActions.createCustodialAccountRequest());
+    api.user.createCustodialAccount(profile, (err, account) => {
+      if (err) {
+        dispatch(syncActions.createCustodialAccountFailure(err));
+      }
+      else {
+        dispatch(syncActions.createCustodialAccountSuccess(account));
```

Comment:
So you're passing in the new `account` but there's no `payload` in the success action... If that's because we're refetching everything (next `async.series`), then perhaps don't include the param (for less confusion) and make a comment?